spec/factories/openingtimes.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :openingtime do
+    opens { "2021-11-29 07:41:48" }
+    closes { "2021-11-29 07:41:48" }
```

Comment:
Why does it store the date, not only the time? 
Also, I would prefer the sample being opened for at least one second ;) 
(Also, maybe add validation that closes - opens is > 0)

Reply:
Update: Only validation still missing